tasks.gradle

```diff
@@ -22,6 +22,48 @@ ext.envFile = { ->
     return [properties: properties, envFile: envFile]
 }
 
+/**
+ * Retrieves legacy properties from a configuration file located in the project's config directory.
+ * The configuration file is expected to start with "Openb" and end with ".properties".
+ *
+ * @return A Properties object containing the properties from the configuration file.
+ *         If the file does not exist, an empty Properties object is returned.
+ */
+def getLegacyProperties = { ->
+    def configFile = file("$projectDir/config/").listFiles().find { it.name.startsWith("Openb") && it.name.endsWith(".properties") }
```

Comment:
Openbravo.properties may be it's the filename more hardcoded in Openbravo and Etendo history, you must try to get it using the exactly file name and path.

Reply:
At one point I thought I heard that the file name could not be hardcoded. I did it and was going to look it up and forgot it! I talked it over with @sebastianbarrozo  and corrected it.